extensions/spring-xd-extension-gemfire/src/main/java/org/springframework/xd/gemfire/GemfireHostPortMixin.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2015 the original author or authors.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.xd.gemfire;
+
+import javax.validation.constraints.AssertTrue;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.util.StringUtils;
+import org.springframework.xd.module.options.spi.ModuleOption;
+import org.springframework.xd.module.options.spi.ProfileNamesProvider;
+
+/**
+ * @author David Turanski
+ */
+public class GemfireHostPortMixin implements ProfileNamesProvider {
+	private String host;
+
+	private String port;
+
+	private boolean useLocator;
+
+	public String getHost() {
+		return host;
+	}
+
+	@ModuleOption("host name of the cache server or locator (if useLocator=true). May be a comma delimited list.")
+	public void setHost(String host) {
+		this.host = host;
+	}
+
+	public String getPort() {
+		return port;
+	}
+
+	@ModuleOption("port of the cache server or locator (if useLocator=true). May be a comma delimited list.")
```

Comment:
Need to remove the period at the end of the description. ModuleOptionsMetadataSanityTests tagged this on build. Will fix on merge.